src/SharpLinks.cs

```diff
@@ -315,11 +346,11 @@ public IContext Finish(IContext context) {
         }
 
         public IContext SaveCacheFile(IContext context) {
-            if (context.Flags.MemCache)
-                return context;
-            // 15. Program exit started. Save the cache file
+            // if (context.Flags.MemCache)
```

Comment:
Do we support memcache? (not sure how it would work) If no, should  we remove [this line from readme](https://github.com/SpecterOps/SharpHound/blob/cda95971746bef79dce5dd40829841abfe585bc3/README.md?plain=1#L60)?